src/resources/filters/customnodes/theorem.lua

```diff
@@ -88,6 +88,22 @@ _quarto.ast.add_handler({
   end
 })
 
+local included_typst_theorems = false
+local letted_typst_theorem = {}
+local function ensure_typst_theorems(reftype)
+  if not included_typst_theorems then
+    included_typst_theorems = true
+    quarto.doc.include_text("in-header", "#import \"@preview/ctheorems:1.1.0\": *")
+    quarto.doc.include_text("in-header", "#show: thmrules")
```

Comment:
I think this will be the place where we change the background color of the theorem. We'll have to learn some typst to do it, but I think our theorem should try to match LaTeX's typesetting a bit more, and the light green background is somewhat jarring.

Reply:
We could at least provide customization at the document level using quarto metadata. It'll be a chance to walk you through a new part of the code.